aspnetcore/performance/response-compression.md

```diff
@@ -0,0 +1,167 @@
+---
+title: Response Compression Middleware | Microsoft Docs
+author: guardrex
+description: An introduction to response compression with instructions on how to use Response Compression Middleware in ASP.NET Core applications.
+keywords: ASP.NET Core, performance, response compression, gzip, accept-encoding, middleware
+ms.author: riande
+manager: wpickett
+ms.date: 01/12/2017
+ms.topic: article
+ms.assetid: de621887-c5c9-4ac8-9efd-f5cc0457a134
+ms.technology: aspnet
+ms.prod: aspnet-core
+uid: performance/response-compression
+---
+# Response Compression Middleware
+
+By [Luke Latham](https://github.com/GuardRex)
+
+[View or download sample code](https://github.com/aspnet/Docs/tree/master/aspnetcore/performance/response-compression/sample)
+
+This document introduces response compression and explains how to use ASP.NET Core Response Compression Middleware. You can explore the features of the Response Compression Middleware with the [sample application](https://github.com/aspnet/Docs/tree/master/aspnetcore/performance/response-compression/sample). The sample illustrates the compression of application responses using Gzip and custom compression providers. It also shows you how to add a MIME type to the default list of MIME types for compression.
+
+## Response Compression
+The time it takes to send content over a network, especially the Internet and to low bandwidth mobile devices, is often the largest share of the total time it takes to satisfy a client's request. The transmission time frequently exceeds the processing time taken on the client and the server. If you can reduce response payload sizes and thus send less data to clients, you can usually increase the responsiveness of your application, sometimes dramatically. One way to reduce payload sizes is to compress an application's responses. When you cannot use a server-based compression technology, you can use ASP.NET Core Response Compression Middleware to compress responses.
```

Comment:
>The time it takes to send content over a network, especially the Internet and to low bandwidth mobile devices, is often the largest share of the total time it takes to satisfy a client's request.The transmission time frequently exceeds the processing time taken on the client and the server. 

RO sentence. Not sure you even need it, most folks know this. Shorten it up a bit. something like

Network bandwidth is a limited resource. If you can reduce response payload sizes and thus send less data to clients, you can usually increase the responsiveness of your application, sometimes dramatically.

Less is more.